incubator/node-red/image/project/Dockerfile

```diff
@@ -1,39 +1,31 @@
 # Install the app dependencies in a full Node docker image
 FROM node:12
 
-# Install OS updates
```

Comment:
Why have you removed updating the base packages?

Reply:
Ah - just saw the linked to issues. If this is being done across all stacks then I assume you know why you are doing this.

Reply:
The intent is to not make version changes in the stack so we can get predictable/consistent stack builds. We have added docs related to this in stack stability levels on our website here. https://appsody.dev/docs/stacks/stacks-overview#stable

Do not introduce any version changes to the content provided by the parent container images (for example: yum upgrade, apt-get dist-upgrade, npm audit fix). If a package contained in the parent image is out of date, contact its maintainers or update it individually.